### PR TITLE
Feature/refactor auth handler

### PR DESCRIPTION
**Pull request by modulo11**
Second part of the basic auth refactoring.

### CODE REVIEW

File: src/main/java/de/retest/recheck/auth/AuthenticationHandler.java [hunk 1]
```diff
@@ -6,8 +6,10 @@
 
 	void showWebLoginUri( URI loginUri );
 
-	void authenticated();
+	void loginPerformed( String token );
```
Comment (roesslerj) [inline src/main/java/de/retest/recheck/auth/AuthenticationHandler.java:9]: Do you have a branch in place for that in downstream projects (read review)?

File: src/main/java/de/retest/recheck/auth/RetestAuthentication.java [hunk 2]
```diff
@@ -37,7 +37,7 @@
 	public static final String AUTH_SERVER_PROPERTY = "de.retest.auth.server";
 	private static final String AUTH_SERVER_PROPERTY_DEFAULT = "https://sso.prod.cloud.retest.org/auth";
 	public static final String RESOURCE_PROPERTY = "de.retest.auth.resource";
-	private static final String RESOURCE_PROPERTY_DEFAULT = "review";
+	private static final String RESOURCE_PROPERTY_DEFAULT = "marvin";
```
Comment (roesslerj) [inline src/main/java/de/retest/recheck/auth/RetestAuthentication.java:40]: Do we make sure in downstream projects to change that (i.e. change the default)?

File: src/main/java/de/retest/recheck/auth/AuthenticationHandler.java (hunk 1 above)
Comment (modulo11) [reply src/main/java/de/retest/recheck/auth/AuthenticationHandler.java:9]: Yes, I do.

File: src/main/java/de/retest/recheck/auth/RetestAuthentication.java (hunk 2 above)
Comment (modulo11) [reply src/main/java/de/retest/recheck/auth/RetestAuthentication.java:40]: This is/can be overwritten (even in the `Rehub` class).

**Review by roesslerj — APPROVED** [review]

**Pull request merged**